packages/handlersjs-http/lib/servers/node/node-http-request-response.handler.spec.ts

```diff
@@ -196,6 +213,28 @@ describe('NodeHttpRequestResponseHandler', () => {
 
     });
 
+    it('should set the charset to utf-8 if no content-type header is specified', async () => {
+
+      const body = 'This is a response body with a certain length.';
+      nestedHttpHandler.handle = jest.fn().mockReturnValueOnce(of({ body, headers: { 'content-type': 'text/html;' }, status:200 }));
+
+      await handler.handle(streamMock).toPromise();
+
+      expect(res.writeHead).toHaveBeenCalledWith(200, { 'content-length': Buffer.byteLength(body, 'utf-8').toString(), 'content-type': 'text/html;' });
+
+    });
+
+    it('should return a undefined body if the body type is not string', async () => {
```

Comment:
This is weird if it succeeds... non-string body types should be JSON stringified

Reply:
The test does succeed however.

Reply:
Then we should look into that ...

Reply:
JSON parsing 1234 .toString() only works with application/json though. It results in a syntax error with application/x-www-form-urlencoded.

Reply:
My bad, wrong description.